build.assets/robotest_run_suite.sh

```diff
@@ -5,19 +5,29 @@ readonly UPGRADE_FROM_DIR=${1:-$(pwd)/../upgrade_from}
 
 # UPGRADE_MAP maps gravity version -> list of OS releases to upgrade from
 declare -A UPGRADE_MAP
-# latest patch release on compatible LTS, keep these up to date
-UPGRADE_MAP[5.5.37]="ubuntu:16"
-UPGRADE_MAP[5.2.16]="ubuntu:16"
 
-# via intermediate upgrade
-UPGRADE_MAP[5.0.35]="ubuntu:16"
+# latest patch release on this release, keep this up to date
+UPGRADE_MAP[5.5.37]="ubuntu:18"
+
+# latest patch release on previous compatible LTS, keep this up to date
+UPGRADE_MAP[5.2.16]="ubuntu:16"
 
-# important versions in the field
-UPGRADE_MAP[5.2.12]="ubuntu:16"
-UPGRADE_MAP[5.5.19]="ubuntu:16"
-UPGRADE_MAP[5.5.20]="ubuntu:16"
-UPGRADE_MAP[5.5.28]="ubuntu:16"
-UPGRADE_MAP[5.5.36]="ubuntu:16"
+# first release from this release & last LTS, these don't need to change until a new major/minor release
+# centos/rhel chosen just for variety
+UPGRADE_MAP[5.5.0]="redhat:7"
+UPGRADE_MAP[5.2.0]="centos:7"
+
+# via intermediate upgrade, debian for variety
+UPGRADE_MAP[5.0.35]="debian:9"
```

Comment:
Same as above, I'm pretty sure this will fail OS validation precheck.

https://github.com/gravitational/gravity.e/blob/version/5.0.x/assets/telekube/resources/app.yaml#L81

Reply:
Will fix.  I got `debian:9` from https://github.com/gravitational/gravity/commit/dabcf701eaa974c3521d8b4caab4a4a157dfba6d#diff-c5171427a28202f7b51fd52a1ddfb760R11 which added debian 9 for 5.0.35.  Admittedly, we don't run nightly builds on 5.5.x currently (though I hope to change that) so I was copying from untested code.

Reply:
debian:8 in 5d3f294.